src/thor/unidirectional_astar.cc

```diff
@@ -210,8 +210,10 @@ inline bool UnidirectionalAStar<expansion_direction, FORWARD>::ExpandInner(
   // Skip shortcut edges for time dependent routes, if no access is allowed to this edge
   // (based on costing method)
   uint8_t restriction_idx = kInvalidRestriction;
+  bool is_dest =
+      destinations_percent_along_.find(meta.edge_id) != std::cend(destinations_percent_along_);
```

Comment:
i assume this was the main case that you were worried about when you implemented this? time dependent unidirectional routing? does constantly hitting this unordered_map cost anything in performance? also why use `std::cend`, please just use `destinations_percent_along_.cend()`

Reply:
Was important to support the same behaviour for all algorithms. There are some changes in recost_forward function for bidirectional routing.
I will fix std::cend -> .cend().